spring-cloud-dataflow-skipper-docs/pom.xml

```diff
@@ -39,6 +39,12 @@
 			<artifactId>json-path</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>io.projectreactor</groupId>
+			<artifactId>reactor-core</artifactId>
+			<version>3.1.8.RELEASE</version>
+			<scope>test</scope>
+		</dependency>
```

Comment:
why is this here?

Reply:
This was needed to have Skipper related doc tests using the specific reactor version that is used by Skipper 1.1.0.